taccsite_cms/static/site_cms/css/src/site.header.css

```diff
@@ -9,7 +9,7 @@
 
 /* NOTE: This is already imported in `site.css` */
 /* TODO: Find a way to be modular, but not redundant */
-@import url("_imports/settings/props.font.css");
+@import url("../../../site_shared/css/src/_imports/settings/font.css");
```

Comment:
**Request for Forgiveness**
I don't like this long relative path. Whenever I next tweak [the PostCSS config](https://github.com/TACC/Core-CMS/blob/main/conf/css/postcss.config.js), I'll set a root path (and update all the paths).